### ISSUE TITLE
Feature Request: 2 2600 Controllers on 1 Gamepad

### ISSUE THREAD

**Issue by KremlingKuthroat19** [MiSTer-devel/Atari7800_MiSTer#33]
There are 8 Atari 2600 games that are single player but require both gamepads to play. Since the 2600 controller has only 1 button, this feature request is just asking if it's possible to add an option to map both controllers' joysticks to the left and right stick, and allow player 1 and player 2's fire buttons to be used as button 1 and either button 2 on the 7800 controller or the paddle button.

Here's the list of games that require 2 controllers:
-Radar Lock
-Raiders of the Lost Ark
-Riddle of the Sphinx
-Solaris
-Spy Hunter
-Star Voyager
-Stargate
-Video Life

Obviously, this would be just a nice-to-have QOL improvement, and not a must have. Could this be possible?

**Comment by Kitrinx** [comment]
I can't remember, but isn't this how the robotron controller works?

**Comment by birdybro** [comment]
Yeah Robotron on 7800 has two options, one to play with one joystick and you fire in the direction you are moving or two joysticks and you can do it independently.

http://www.8-bitcentral.com/reviews/7800robotron.html

**Comment by KremlingKuthroat19** [comment]
> I can't remember, but isn't this how the robotron controller works?

It almost works perfectly. When Port 1 and Port 2 are selected as Robotron, you can move both analog sticks independently on one controller. Raiders of the Lost Ark allows you to move Indy with the right stick and select the inventory with the left stick. The only issue is you need to use Port 2 for the 2nd fire button which a lot of the games listed use. I just tested some of the games and here's a breakdown of what the 2nd fire button does:

-Raiders of the Lost Ark (2nd button starts game)
-Solaris (2nd button opens up map screen)
-Stargate (2nd buttons acts as a bomb)
-Video Life (2nd button confirms a placement)

It's not that big of a deal, but just asking if the code for the Robotron controller can be modified to make either the 2nd action button on the 7800 controller or the paddle button act as the 2nd fire button so these games can be played with one controller?
